pkg/input/input.go

```diff
@@ -39,3 +41,7 @@ func FromFile(file string) (string, error) {
 
 	return "", nil
 }
+
+func FromURL(url string) (string, error) {
```

Comment:
It seems that this function isn't necessary and callers can just call `loader.ContentFromURL`.

Reply:
I mainly wanted them all of these to exist in the same package. Thoughts on that?

Reply:
Not something I think is valuable, but feel free to keep it.